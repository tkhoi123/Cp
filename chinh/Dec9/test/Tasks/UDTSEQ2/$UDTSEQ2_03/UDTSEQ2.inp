5 1
2 3 8588
