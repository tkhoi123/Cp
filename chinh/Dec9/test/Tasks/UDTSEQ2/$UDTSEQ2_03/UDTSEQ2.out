0 8588 17176 0 0 